### PR TITLE
added content to troubleshooting guide

### PR DESCRIPTION
**Pull request by shaynacohen**

### CODE REVIEW

**Comment by shaynacohen** [comment]
Note: there are a couple of links to community posts that are not public (internal community posts), but I'm hoping no one will encounter these while we fill in the details over the next couple of weeks.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/c562aef2893eaaf1a631810b810fd7a0f5d0d513/index.html)

File: docs/en/quick-answers/troubleshooting.md [hunk 1]
```diff
@@ -10,23 +10,78 @@ Troubleshooting
...
+- [Failed to change work area - Error during execution of MEL script](https://community.shotgridsoftware.com/t/new-file-maya-action-error/8225)
+- [‘ascii’ codec can’t decode byte 0x97 in position 10: ordinal not in range](https://community.shotgridsoftware.com/t/ascii-problem/7688)
+- [TankError: Tried to resolve a path from the template](https://community.shotgridsoftware.com/t/error-when-resolving-path-from-template-at-new-file-open-missing-fields/6468)
+- [ERROR 18:13:28.365:Hiero(34236): Error! Task type](https://community.shotgridsoftware.com/t/cant-get-shotgun-toolkit-to-work-with-nuke-studio-config-default2/4586)
+- [Cannot find procedure "MTsetToggleMenuItem"](https://community.shotgridsoftware.com/t/tk-maya-cannot-find-procedure-mtsettogglemenuitem/4629)
+- [Could not resolve row id for path!](https://community.shotgridsoftware.com/t/how-to-troubleshoot-folder-creation-errors/3578)
+- [Database concurrency problems: The path <PATH> is already associated with {% include product %} entity <ENTITY>](https://community.shotgridsoftware.com/t/how-to-troubleshoot-folder-creation-errors/3578)
```
Comment (shaynacohen) [inline docs/en/quick-answers/troubleshooting.md:56]: @drewavisautodesk any tips on how to get this to not escape the error message content? <>

Comment (drewavisautodesk) [reply docs/en/quick-answers/troubleshooting.md:56]: You could try putting them in backticks: `<ENTITY>`

Or if that doesn't work, use HTML entities: &lt;ENTITY&gt;

Comment (drewavisautodesk) [reply docs/en/quick-answers/troubleshooting.md:56]: Of course neither of those actually show the markdown I used... I'll send you them in slack

Comment (shaynacohen) [reply docs/en/quick-answers/troubleshooting.md:56]: Thanks!

Comment (drewavisautodesk) [reply docs/en/quick-answers/troubleshooting.md:56]: I wonder if the {% include product %} in the link text will resolve correctly?  I was having issues with the include working in links/headings

**Review by drewavisautodesk — APPROVED** [review]
Preview looks good, links are working for me.  My one thought is that it might be nice to organize all the error codes under their own heading, alphabetically

**Comment by shaynacohen** [comment]
> Preview looks good, links are working for me. My one thought is that it might be nice to organize all the error codes under their own heading, alphabetically

Yeah, I like that idea!

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/59c38c4c2c910020e6e834dfa44a4cac6156a31e/index.html)

**Comment by shaynacohen** [comment]
@drewavisautodesk I reorganized a bit, did some general alphabetizing, and added the community gif at the bottom. Let me know what you think :)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/0bad9d2844985505116c551b740a1f6933582475/index.html)

**Pull request merged**